src/hotspot/share/opto/graphKit.cpp

```diff
@@ -1019,15 +1019,20 @@ bool GraphKit::compute_stack_effects(int& inputs, int& depth) {
     code = method()->java_code_at_bci(bci() + 1);
   }
 
-  BasicType rtype = T_ILLEGAL;
-  int       rsize = 0;
-
   if (code != Bytecodes::_illegal) {
     depth = Bytecodes::depth(code); // checkcast=0, athrow=-1
+  }
+
+  auto rsize = [&]() {
+    BasicType rtype = T_ILLEGAL;
+    int sz = 0;
+
     rtype = Bytecodes::result_type(code); // checkcast=P, athrow=V
     if (rtype < T_CONFLICT)
-      rsize = type2size[rtype];
-  }
+      sz = type2size[rtype];
```

Comment:
[pre-existing] Missing braces around the consequent.  Maybe write this instead as
```
if (rtype < T_CONFLICT) {
  return type2size[rtype];
} else {
  return 0;
}
```
avoiding the introduction of the `sz` variable.

Reply:
make sense. I should be an expression.